asmcomp/power/arch.ml

```diff
@@ -115,3 +115,11 @@ let print_specific_operation printreg op ppf arg =
         printreg arg.(0) printreg arg.(1) printreg arg.(2)
   | Ialloc_far { bytes; _ } ->
       fprintf ppf "alloc_far %d" bytes
+
+(* Specific operations that are pure *)
+
+let operation_is_pure _ = true
```

Comment:
Ialloc_far is missing, here and can_raise.

Reply:
Right.  The issue is cosmetic, because `Ialloc_far` is introduced very late, in the branch relaxation pass of Emit, and the `is_pure` and `can_raise` tests are not used then.  But it's clearer to document the behavior of `Ialloc_far`, indeed.  I did that already for ARM64.